docs/faq.rst

```diff
@@ -6,7 +6,11 @@ Frequently Asked Questions
 
 In case you have a general question that is not answered here, consider submitting a `new issue`_.
```

Comment:
Should we rather refer users to the new *Discussions* feature of Github instead of Issues?

Reply:
I will also add it directly when merging to master, since it does not interfere with the CI.